css/src/components/form/input.scss

```diff
@@ -72,4 +80,65 @@ $input-focus-box-shadow-size: 0 0 0 0.125rem !default;
 			box-shadow: $input-focus-box-shadow-size $input-success-border-color;
 		}
 	}
+
+	&.input-icon-right {
```

Comment:
![image](https://user-images.githubusercontent.com/30843002/178315584-f3ea4220-76b6-4e38-bd4a-f34521c20b36.png)

In the example, I'm still seeing an indented cursor when the icon is on the right.

Reply:
Should be removed now